jobs/jumpbox/templates/bin/watcher

```diff
@@ -227,7 +227,10 @@ EOF
 	done
 
 	mkdir -p ${JUMPBOX_ETC}
-	cp -a /etc/passwd /etc/*shadow /etc/group ${JUMPBOX_ETC}/
+	if [ "$(ls -A ${JUMPBOX_ETC})" ]; then
+            cp -a /etc/passwd /etc/*shadow /etc/group ${JUMPBOX_ETC}/
+        fi
```

Comment:
JUMPBOX_ETC isn't the problem, the problem is that /etc/* files are being overwritten by the content of JUMPBOX_ETC. According to https://github.com/cloudfoundry-community/jumpbox-boshrelease/commit/00c52ac071d6a023f1dca451bad0e5ef8eb43d4f#diff-829738c46d60e973314befcaef107769R11,  overwrites were being done to preserve UID/GIDs across VM recreation (a feature we want). But it has a side-effect of not letting anything be modified to users/groups outside the realm of the jumpbox script (which we probably dont want, since bosh-ssh creates users that the watcher script will remove while they are logged in)